test/unit/views/PhaseBannerSpec.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import base.Injector
+import play.api.test.FakeRequest
+import play.twirl.api.Html
+import views.html.components.gds.phaseBanner
+
+class PhaseBannerSpec extends ViewSpec with Injector {
+
+  private val page = instanceOf[phaseBanner]
```

Comment:
Nit: This is not a page really ;-) I'd rather call it `banner` for e.g. But this is not a big deal.